### PR TITLE
Update ACM must gather in 2.6

### PR DESCRIPTION
**Pull request by oafischer**
https://github.com/stolostron/backlog/issues/25475

### CODE REVIEW

File: troubleshooting/must_gather.adoc [hunk 1]
```diff
@@ -63,3 +63,14 @@ IMAGE=$REGISTRY/rhacm2/acm-must-gather-rhel8@sha256:ff9f37eb400dc1f7d07a9b6f2da9
...
 oc adm must-gather --image=$IMAGE --dest-dir=./data
 ----
+
+. Run the following commands to collect all the information:
+
+----
+REGISTRY=<internal.repo.address:port>
+IMAGE1=$REGISTRY/rhacm2/acm-must-gather-rhel8:v2.5
+IMAGE2=$REGISTRY/multicluster-engine/must-gather-rhel8:v2.0
+
```
Comment (zhiweiyin318) [inline troubleshooting/must_gather.adoc:73]: not sure the image tag should be v2.6 and v2.1 in 2.6 release?

Comment (oafischer) [reply troubleshooting/must_gather.adoc:73]: @cadawson Will v2.6 and v2.1 be available in the 2.6 release?

Comment (cadawson) [reply troubleshooting/must_gather.adoc:73]: I am not sure, but I would think that is logical.

**Review by cadawson — APPROVED** [review]
Looks good. Thanks!

File: troubleshooting/must_gather.adoc (hunk 1 above)
Comment (oafischer) [reply troubleshooting/must_gather.adoc:73]: OK since you both have a hunch that it will be available, I'll go with 2.6 and 2.1. Thanks.

**Pull request merged**